components/infobox/commons/infobox.lua

```diff
@@ -136,6 +136,10 @@ function Infobox:_createChronologyRow(previous, next)
         previousWrapper :node(previousArrow)
                         :wikitext('&nbsp;[[' .. previous .. ']]')
 
+        node:node(previousWrapper)
+    elseif doesNextExist then
+        local previousWrapper = mw.html.create('div'):addClass('infobox-cell-2')
```

Comment:
We need it in both branches so init before the if

Reply:
right, we check for them both being false before, so yeah before the ifs is better
and can also use `else` instead of `elseif`